src/renderer/coremods/commands/commands.ts

```diff
@@ -0,0 +1,382 @@
+          ],
+        };
+      } catch (err) {
+        return {
+          send: false,
+          embeds: [
+            {
+              type: "rich",
+              color: 0xdd2d2d,
+              title: "Error",
+              description: err as string,
+            },
+          ],
+        };
+      }
+    },
+  },
+  {
+    name: "reload",
+    description: "Reload a plugin or theme",
+    usage: "/reload <plugin/theme>",
+    options: [
+      {
+        name: "addon",
+        description: "Choose the addon you want to reload",
+        type: ApplicationCommandOptionType.String,
+        required: true,
+        get choices() {
+          let choices = [];
+
+          const enabledPlugins = Array.from(plugins.plugins.values());
+          const enabledThemes = Array.from(themes.themes.values());
+
+          choices.push(
+            ...enabledPlugins.map((plugin) => ({
+              name: plugin.manifest.name,
+              displayName: `Plugin: ${plugin.manifest.name}`,
+              value: plugin.manifest.id,
+            })),
+          );
+          choices.push(
+            ...enabledThemes.map((theme) => ({
+              name: theme.manifest.name,
+              displayName: `Theme: ${theme.manifest.name}`,
+              value: theme.manifest.id,
+            })),
+          );
+
+          return choices;
+        },
+      },
+    ],
+    executor: async (args) => {
+      try {
+        if (plugins.plugins.has(args[0].value as string)) {
+          await plugins.reload(args[0].value as string);
+        } else {
+          themes.reload(args[0].value as string);
+        }
+        return {
+          send: false,
+          embeds: [
+            {
+              type: "rich",
+              color: 0x1bbb1b,
+              title: "Success",
+              description: `${args[0].value} has been reloaded!`,
+            },
+          ],
+        };
+      } catch (err) {
+        return {
+          send: false,
+          embeds: [
+            {
+              type: "rich",
+              color: 0xdd2d2d,
+              title: "Error",
+              description: err as string,
+            },
+          ],
+        };
+      }
+    },
+  },
+  {
+    name: "list",
+    description: "List all plugins/themes",
+    usage: '/list <true/false> <"plugin"/"theme"> <>',
+    options: [
+      {
+        name: "send",
+        description: "Whether you want to send this or not.",
+        type: ApplicationCommandOptionType.Boolean,
+        required: false,
+      },
+      {
+        name: "addon type",
+        description: "Choose what type of addons to list.",
+        type: ApplicationCommandOptionType.String,
+        required: true,
+        choices: [
+          {
+            name: "Theme",
+            displayName: `List themes`,
+            value: "theme",
+          },
+          {
+            name: "Plugin",
+            displayName: `List plugins`,
+            value: "plugin",
+          },
+        ],
```

Comment:
- Rename to just `type`
- Make optional
- Add option for "both", which is the default if none is selected.